auraescript/LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
why did this get deleted?

Reply:
By accident, but I'm not sure if we still need it. Is this an artifact of when we had multiple repos. Put back for now.